libs/component-library/src/lib/tabs/tabs.tsx

```diff
@@ -0,0 +1,145 @@
+import { useTheme } from '@shopify/restyle';
+import React from 'react';
+import { FxBox, FxReanimatedBox } from '../box/box';
+import { FxPressableOpacity } from '../pressable-opacity/pressableOpacity';
+import { FxText, FxTextProps } from '../text/text';
+import { FxTheme } from '../theme/theme';
+import {
+  useAnimatedReaction,
+  useAnimatedStyle,
+  useDerivedValue,
+  useSharedValue,
+  withTiming,
+} from 'react-native-reanimated';
+
+const ANIMATION_DURATION = 300;
+const SELECTED_TEXT: FxTextProps = { color: 'greenBase' };
+const PRESSED_TEXT: FxTextProps = { color: 'backgroundSecondary' };
+
+type FixedTabProps = {
+  text: string;
+  selected: boolean;
+  onPress: () => void;
+};
+
+const FixedTab = ({ text, selected, onPress }: FixedTabProps) => {
+  const theme = useTheme<FxTheme>();
+  const [pressed, setPressed] = React.useState<boolean>(false);
+
+  let textOverrides = pressed ? PRESSED_TEXT : {};
+  textOverrides = { ...textOverrides, ...(selected ? SELECTED_TEXT : {}) };
+
+  return (
+    <FxPressableOpacity
+      flex={1}
+      onPress={onPress}
+      alignItems="center"
+      justifyContent="center"
+      borderBottomWidth={2}
+      borderBottomColor="content3"
+      paddingVertical="16"
+      disabled={selected}
+      hitSlop={0}
+      onPressIn={() => {
+        setPressed(true);
+      }}
+      onPressOut={() => {
+        setPressed(false);
+      }}
+      style={(args) => [
+        args.pressed && {
+          opacity: 1,
+          backgroundColor: theme.colors.backgroundPrimary,
+          borderBottomColor: theme.colors.backgroundSecondary,
+        },
+      ]}
+    >
+      <FxText variant="bodySmallRegular" color="content3" {...textOverrides}>
+        {text}
+      </FxText>
```

Comment:
As discussed in huddle, it might be good for the text to animate too.

Reply:
I'm going to look more at this in a separate PR. I played around with it for a bit and was having some difficulties getting it where I wanted it.